example_omitempty_test.go

```diff
@@ -0,0 +1,59 @@
+package sqlair_test
+
+import (
+	"database/sql"
+	"fmt"
+
+	"github.com/canonical/sqlair"
+)
+
+func Example() {
```

Comment:
Might be best to call this `Example_omitempty` so that its clear this is not a whole package example. It should still be recognised if I am interpreting the [blog post](https://go.dev/blog/examples) correctly

Reply:
I will include it to further differentiate that this example is about `omitempty` but it is still counted as a whole package example. I tested it locally using `godoc` with the name change.